subfun/BIDS_RFX.m

```diff
@@ -22,14 +22,17 @@ function BIDS_RFX(action, mmFunctionalSmoothing, mmConSmoothing, opt)
 % The Contrast names should match those in the single level FFX and in THE
 % SAME ORDER.
 
+% Get the current working directory
+WD = pwd;
+
 % if input has no opt, load the opt.mat file
 if nargin<4
-    load('opt.mat') 
+    cd ..
+    load('opt.mat')
     fprintf('opt.mat file loaded \n\n')
 end
 
-% Get the current working directory
-WD = pwd;
+cd(WD)
 
```

Comment:
I go back to the WD (scripts directory) after the opt.mat is loaded from the above folder.